src/raft.c

```diff
@@ -306,6 +306,26 @@ void RaftExecuteCommandArray(RedisRaftCtx *rr,
                              RaftRedisCommandArray *cmds)
 {
     RedisModuleCallReply *reply;
+    RedisModuleUser *user = NULL;
+
+    if (cmds->acl) {
+        int nokey;
+        user = RedisModule_DictGet(rr->acl_dict, cmds->acl, &nokey);
+        if (nokey) {
+            char user_name[64];
+            uint64_t count = RedisModule_DictSize(rr->acl_dict);
+            count++;
+            snprintf(user_name, 64, "redis_raft%lu", count);
+            user = RedisModule_CreateModuleUser(user_name);
+            RedisModule_Assert(user != NULL);
+
+            size_t acl_len;
+            const char *acl_str = RedisModule_StringPtrLen(cmds->acl, &acl_len);
```

Comment:
```suggestion
            const char *acl_str = RedisModule_StringPtrLen(cmds->acl, NULL);
```

we can ignore acl size

Reply:
removed, as unused